xqueue_watcher/client.py

```diff
@@ -174,7 +183,7 @@ def run(self):
                     break
         while self.running:
             if not self.process_one():
-                time.sleep(1)
+                time.sleep(self.wait_duration)
```

Comment:
Should this be renamed to something like `self.poll_interval`?

Reply:
Yup, much better name